scripts/apps/web-publisher/styles/_helpers_variables.sass

```diff
@@ -0,0 +1,45 @@
+$superdesk-green:       rgba(49,172,102,1);
+
+// Grays
+// -------------------------
+$black:                 #000;
+$grayDarker:            #222;
+$grayDark:              #333;
+$gray:                  #666;
+$grayLight:             #999;
+$grayLighter:           #F5F5F5;
+$white:                 #fff;
+
+// Accent colors
+// -------------------------
+$blue:                  #5d9bc0;
+$blueDark:              #0064cd;
+$avatarBlue:            #005b7f;
+$green:                 #46a546;
+$red:                   #E51C23;
+$yellow:                #ffc40d;
+$orange:                #f89406;
+$purple:                #9c5e96;
+
+$sd-background:         #f8f8f8;
+$sd-text:               $grayDark;
+$sd-blue:               #5ea9c8;
+$sd-blue-text:          #3599b6;
+$sd-hover:              #eff7fa;
+$sd-keyword:            #216278;
+$sd-highlight:          #9b59b6;
+
+$label-light:           #dedede;
+$label-medium:          #bababa;
+$label-prime:           #61a9c6;
+$label-snap:            #9761A0;
+
+$sd-text-label:         #909090;
+
+
+$todo-color :           #b70000;
+$inprogress-color :     #cc7800;
+$done-color :           #288001;
+
+.contentPadding
+  padding: 20px
```

Comment:
This syntax without the curly braces is quite cool. We've not used it elsewhere in the project but I'd be happy to adopt it!

Although, we do use 4 space indentation instead of 2, so please amend that.